### ISSUE TITLE
--appending flag

### ISSUE THREAD

**Issue by hentheor** [zeFresk/deep-position-analysis#1]
Hi

It appears that --appending flag does nothing. 

Also in a multgame pgn file, iterating process seems to freeze due to faulty nps evals (or cache?)
If I recreate .cache-db after each iter completes, process continues as it should...

Can you please take a look?
Thank you

M.N.

**Comment by zeFresk** [comment]
Hello !
Thank you for your report, I didn't expect anyone to use my script !

Since I didn't expected anyone to use the master branch I disabled the bug-catching.
This is currently not even in alpha, that's why the --appending option isn't working anymore since I added cache. I will try to add it ASAP :)

However the second problem your describing seems to be a genuine bug so thank you for reporting it !!

I will submit a fix ASAP.

Thank you very much for submitting an issue !
